bindings/matrix-sdk-ffi/src/timeline/configuration.rs

```diff
@@ -82,4 +139,26 @@ pub struct TimelineConfiguration {
 
     /// How often to insert date dividers
     pub date_divider_mode: DateDividerMode,
+
+    /// Should the read receipts and read markers be tracked for the timeline
+    /// items in this instance?
+    ///
+    /// As this has a non negligible performance impact, make sure to enable it
+    /// only when you need it.
+    pub track_read_receipts: bool,
```

Comment:
Just to confirm, looking at the previous code it looks like this should be always false for now and it should only come in play when we get rid of the `init_timeline` method, right?

Reply:
Exactly!